.travis.yml

```diff
@@ -21,4 +21,11 @@ after_script: greenkeeper-lockfile-upload
 
 after_success:
   - cat ./coverage/lcov.info | ./node_modules/codecov.io/bin/codecov.io.js
-  - cat ./coverage/lcov.info | ./node_modules/coveralls/bin/coveralls.js
\ No newline at end of file
+  - cat ./coverage/lcov.info | ./node_modules/coveralls/bin/coveralls.js
+
+deploy:
+  provider: npm
+  email: $NPM_DEPLOY_EMAIL
+  api_key: $NPM_DEPLOY_API_KEY
+  on:
+    branch: release
```

Comment:
The variables have been added...

<img width="865" alt="screen shot 2018-03-22 at 18 10 49" src="https://user-images.githubusercontent.com/2536442/37789648-68998c1c-2dfc-11e8-9224-2b053affd6dd.png">

Reply:
I think we can prepare releases on a release branch but then merge it to `master` which will trigger a deployment.
Also, there has to be a process in place to bump the versions up, otherwise it will fail to deploy.